python/ola/MACAddressTest.py

```diff
@@ -27,6 +28,15 @@
 
 class MACAddressTest(unittest.TestCase):
 
+  def allNotEqual(self, t):
+    for pair in itertools.combinations(t, 2):
+      self.assertNotEqual(pair[0], pair[1])
+
+  def allHashNotEqual(self, t):
+    h = map(hash, t)
+    for pair in itertools.combinations(h, 2):
+      self.assertNotEqual(pair[0], pair[1])
```

Comment:
Given these two functions are used in lots of the cases, can you pull them out into a TestUtils or something file and just import that each time, like we do in C++ land.

Reply:
Yeah, I think I even had it that way initially and got tripped up by the distcheck setup.  I think I figured that out later, so will move back to that setup.

Reply:
Moved to python/TestUtils.py